### DISCUSSION TITLE
How to use Appium Inspector 2021.9.1?

### DISCUSSION THREAD

**Discussion by santtyamen** [appium/appium#15890] [Q&A]
Hi, who help me for to use Appium Inspector 2021.9.1
<img width="1277" alt="Captura de Pantalla 2021-09-24 a la(s) 12 15 26" src="https://user-images.githubusercontent.com/67275727/134715187-6a7bf692-8964-4f7a-a035-600446f3cbaa.png">

**Comment by KazuCocoa** [comment]
To which server?
I guess you should add `/wd/hub` as Remote Path if you ran appium 1.X.

**Reply by santtyamen** [reply]
I thought these variables are set automatically like Appium Desktop.
:D
![Captura de Pantalla 2021-09-28 a la(s) 8 57 55](https://user-images.githubusercontent.com/67275727/135102300-8aeb6bd0-b33e-40d1-8d7d-ad145b445b3e.png)

Ready!!

**Reply by jlipps** [reply]
> I thought these variables are set automatically like Appium Desktop.

They are, but the default value is `/` for Appium 2.0, not `/wd/hub`, and Appium Inspector has been updated for Appium 2.0 compatibility.

**Comment by sumitnaidu8** [comment]
<img width="1263" alt="Screenshot 2021-10-04 at 1 41 05 PM" src="https://user-images.githubusercontent.com/91743870/135816330-b2a952c0-9b9d-45cc-9713-77f48c7b6c65.png">

Keep getting timeout issue

**Reply by TimurNurlygayanov** [reply]
is Appium server running on this host?

**Reply by sumitnaidu8** [reply]
> is Appium server running on this host?

Yes, it's running
[Appium] Appium REST http interface listener started on 0.0.0.0:4723

**Reply by jlipps** [reply]
what do the appium logs say?

**Reply by sumitnaidu8** [reply]
[debug] [WD Proxy] Matched '/status' to command name 'getStatus'
[debug] [WD Proxy] Proxying [GET /status] to [GET http://127.0.0.1:8100/status] with no body
[debug] [iProxy@00008030:8100] Connection was refused to port 8100
[WD Proxy] socket hang up
[debug] [WD Proxy] Matched '/status' to command name 'getStatus'
[debug] [WD Proxy] Proxying [GET /status] to [GET http://127.0.0.1:8100/status] with no body
[debug] [iProxy@00008030:8100] Connection was refused to port 8100
[WD Proxy] socket hang up
[debug] [WebDriverAgent] Unable to connect to running WebDriverAgent: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
[WebDriverAgent] Getting status of WebDriverAgent on device timed out. Continuing
[debug] [WD Proxy] Matched '/status' to command name 'getStatus'
[debug] [WD Proxy] Proxying [GET /status] to [GET http://127.0.0.1:8100/status] with no body
[debug] [iProxy@00008030:8100] Connection was refused to port 8100'

**Reply by jlipps** [reply]
Did you try to address that error? It looks like WDA did not launch correctly. This issue has nothing to do with the Inspector. First ensure that you can actually run iOS sessions on this Appium server.
